packages/fogbender/src/floatyWidget.tsx

```diff
@@ -0,0 +1,242 @@
+// Preact version of the SolidJS floaty widget
+import { render } from "preact";
+import { useEffect, useMemo, useRef, useState } from "preact/hooks";
```

Comment:
I think hooks version of preact adds 30% to the bundle, probably fine for this case (compared to twind), but not great idea in general

Reply:
I know an even worse idea - it not working with NextJS :D